io/driver/driver_bind_test.py

```diff
@@ -45,6 +46,10 @@ def setUp(self):
         self.count = int(self.params.get('count', default=1))
         if not self.pci_devices:
             self.cancel("No pci_adresses Given")
+        smm = SoftwareManager()
+        for pkg in ["pciutils"]:
```

Comment:
why for loop, just checking package as string will be good

Reply:
@PraveenPenguin To make it easy the next time we need to add more packages, and also to follow an uniform approach.

Reply:
sure,  it does not look good maybe when it requires to add more package we can have for loop logic